src/packages/sdk/src/cli/getSchema.ts

```diff
@@ -359,11 +370,20 @@ export function getSchemaDirSync(schemaPathFromArgs?: string): string | null {
   return null
 }
 
+// TODO: This should probably return string | null to stay consistent with the other functions
 export function getSchemaSync(schemaPathFromArgs?: string): string {
   const schemaPath = getSchemaPathSync(schemaPathFromArgs)
 
   if (!schemaPath) {
-    throw new Error(`Could not find ${schemaPath || 'schema.prisma'}`)
+    throw new Error(
+      `Either provide ${chalk.greenBright('--schema')} ${chalk.bold(
+        'or',
+      )} configure a path in your package.json in a \`prisma.schema\` field ${chalk.bold(
+        'or',
+      )} make sure that you are in a folder with a ${chalk.greenBright(
+        'schema.prisma',
+      )} file.`,
+    )
```

Comment:
The message here is a bit different and missing the link https://pris.ly/d/prisma-schema-location`, is that expected?